src/hypothesis/extra/typehints.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import print_function
+import sys
+from hypothesis import strategies as st
+from hypothesis.strategies import SearchStrategy
+from typing import Dict, Tuple, List, Iterator, Set, Union, Optional, NamedTuple, Callable, Sequence, Iterable, Generator
+import operator
+from collections import OrderedDict
+compose = lambda f,g: lambda *x: f(g(*x))
+# see https://docs.python.org/3/library/typing.html
+try:
```

Comment:
It's a python3 application, so it requires python 3 to run. It can be used to typecheck python 2 code.